stdlib/runtime/equal.gr

```diff
@@ -8,6 +8,7 @@ include "runtime/unsafe/wasmf32" as WasmF32
 include "runtime/unsafe/tags" as Tags
 include "runtime/numbers" as Numbers
 from Numbers use { isNumber, numberEqual }
+include "runtime/unsafe/memory"
```

Comment:
Could you group the includes a little better? Otherwise looks great.

Reply:
Done